.github/workflows/release.yml

```diff
@@ -64,6 +64,12 @@ jobs:
           git push origin HEAD:main
         if: inputs.draft != true
 
+      - name: Create and push tag
+        if: inputs.draft != true
+        run: |
+          git tag v${{ steps.prepare-release.outputs.next-release-tag }}
+          git push origin v${{ steps.prepare-release.outputs.next-release-tag }}
```

Comment:
you'll want to force push this to bypass branch protection as the automation user

Reply:
and or `--tags` as part of the push command